src/ol/render/canvas/canvasreplay.js

```diff
@@ -1376,6 +1402,16 @@ ol.render.canvas.PolygonReplay.prototype.getBufferedMaxExtent = function() {
 
 
 /**
+ * @param {boolean} rightHanded Whether polygons are assumed to follow
+ *     the right-hand rule.
+ */
+ol.render.canvas.PolygonReplay.prototype.setRightHanded =
```

Comment:
Hm. Now that I look at it again, the whole logic could actually be removed. Because ol.render.Feature will always assume righthanded anyway.